file-formats/sito/type/zif_aff_sito_v1.intf.abap

```diff
@@ -0,0 +1,551 @@
+    BEGIN OF ty_sit2_os_vh_map_s,
+      "! <p class="shorttext">Type</p>
+      "! App for value help
+      "! $required
+      type       TYPE ty_sit2_do_vh_app_type,
+      "! <p class="shorttext">Field Name</p>
+      "! Object field
+      field_name TYPE c LENGTH 30,
+      "! <p class="shorttext">Entity Set</p>
+      "! Name
+      entity_set TYPE c LENGTH 40,
+      "! <p class="shorttext">Property</p>
+      "! Name
+      property   TYPE c LENGTH 40,
+      "! <p class="shorttext">ID</p>
+      "! Service id for value help
+      id         TYPE c LENGTH 32,
+    END OF ty_sit2_os_vh_map_s,
+
+    "! <p class="shorttext">Services</p>
+    "! Value help mapping for situation object structure - service
+    ty_sit2_os_vh_map_s_list TYPE STANDARD TABLE OF ty_sit2_os_vh_map_s WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Texts</p>
+    "! Situation object events text
+    BEGIN OF ty_sit2_obj_event_t,
+      "! <p class="shorttext">Description</p>
+      "! Description
+      description TYPE c LENGTH 255,
+    END OF ty_sit2_obj_event_t,
+
+    "! <p class="shorttext">Texts</p>
+    "! Situation object events text
+    ty_sit2_obj_event_t_list TYPE STANDARD TABLE OF ty_sit2_obj_event_t WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Texts</p>
+    "! Text table for situation object actions
+    BEGIN OF ty_sit2_obj_act_t,
+      "! <p class="shorttext">Name</p>
+      "! Name of action
+      name        TYPE c LENGTH 40,
+      "! <p class="shorttext">Description</p>
+      "! Description of action
+      description TYPE c LENGTH 255,
+    END OF ty_sit2_obj_act_t,
+
+    "! <p class="shorttext">Texts</p>
+    "! Text table for situation object actions
+    ty_sit2_obj_act_t_list TYPE STANDARD TABLE OF ty_sit2_obj_act_t WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Navigations</p>
+    "! Situation object action - navigation
+    BEGIN OF ty_sit2_obj_act_nav,
+      "! <p class="shorttext">ID</p>
+      "! Navigation id
+      id TYPE c LENGTH 32,
+    END OF ty_sit2_obj_act_nav,
+
+    "! <p class="shorttext">Navigations</p>
+    "! Situation object action - navigation
+    ty_sit2_obj_act_nav_list TYPE STANDARD TABLE OF ty_sit2_obj_act_nav WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Callbacks</p>
+    "! Situation object action - callback
+    BEGIN OF ty_sit2_obj_act_cb,
+      "! <p class="shorttext">ID</p>
+      "! Callback id
+      id TYPE c LENGTH 32,
+    END OF ty_sit2_obj_act_cb,
+
+    "! <p class="shorttext">Callbacks</p>
+    "! Situation object action - callback
+    ty_sit2_obj_act_cb_list TYPE STANDARD TABLE OF ty_sit2_obj_act_cb WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Parameters</p>
+    "! Situation object navigation parameter
+    BEGIN OF ty_sit2_obj_nav_par,
+      "! <p class="shorttext">Name</p>
+      "! Name of navigation parameter
+      name TYPE c LENGTH 30,
+    END OF ty_sit2_obj_nav_par,
+
+    "! <p class="shorttext">Parameters</p>
+    "! Situation object navigation parameter
```

Comment:
```suggestion
    "! Situation object navigation parameters
```

Reply:
types for navigation and callback parameters have been combined to reusable type "parameters", as suggested by @schneidermic0